app/validators/prospective_student_validator.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class ProspectiveStudentValidator < ActiveModel::Validator
```

Comment:
`test/validators` dizinine baktığımda `app/validators` dizinindeki validasyonlardan sadece `datetime`'ın test edildiğini görüyorum (11 validasyondan sadece 1 tanesi test edilmiş yani).  Bu validasyon dizini büyüme eğilimi gösteriyor, testlerini eksik etmememiz lazım.  Örneğin bu validasyon için `/test/validators`'a test ekleyelim, zaman içinde diğerlerini de tamamlamaya çalışalım.